roles/openshift_prometheus/templates/prometheus.j2

```diff
@@ -78,7 +78,7 @@ spec:
         - --storage.tsdb.retention=6h
         - --storage.tsdb.min-block-duration=2m
         - --config.file=/etc/prometheus/prometheus.yml
-        - --web.listen-address=localhost:9090
+        - --web.listen-address=0.0.0.0:9090
```

Comment:
We should probably make this address a variable, and default it to 'localhost' as we have it currently.